skyfield/iokit.py

```diff
@@ -51,7 +51,7 @@ def _filename_of(url):
 _IERS = 'https://hpiers.obspm.fr/iers/bul/bulc/'
 _JPL = 'ftp://ssd.jpl.nasa.gov/pub/eph/planets/bsp/'
 _NAIF = 'https://naif.jpl.nasa.gov/pub/naif/generic_kernels/spk/satellites/'
-_USNO = 'http://maia.usno.navy.mil/ser7/'
```

Comment:
[IERS document](https://datacenter.iers.org/data/latestVersion/6_BULLETIN_A_V2013_016.txt) says the server at https://maia.usno.navy.mil/ser7 should return to life on 30 April, 2020, but with HTTPS activated. You could just leave this constant here, so at the time the server returns we can just re-switch to it… WDYT?